### ISSUE TITLE
Replay button does not have correct aria-label

### ISSUE THREAD

**Issue by agliga** [google/shaka-player#3512]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
3.1.1

**Can you reproduce the issue with our latest release version?**
Yes

**Can you reproduce the issue with the latest code from `master`?**
Yes

**Are you using the demo app or your own custom app?**
Yes

**If custom app, can you reproduce the issue using our demo app?**
Yes

**What browser and OS are you using?**
Any

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**
N/A

**What are the manifest and license server URIs?**

N/A

**What configuration are you using?  What is the output of `player.getConfiguration()`?**

N/A

**What did you do?**

When the video ends, I expect the replay button to have aria-label of `replay` not play, to denote to users that the video has ended and clicking this button will replay the video from the start
https://github.com/google/shaka-player/blob/4c975621a0110a7e6d21be2b8e2cebeb80e1cb1c/ui/small_play_button.js#L37-L45 This needs to also have aria-label changes, or to update the `updateAriaLabel` method.

**What did you expect to happen?**
The aria-label should be `replay`

**What actually happened?**

The aria-label stays as `play`
<img width="324" alt="Screen Shot 2021-07-07 at 11 57 21 AM" src="https://user-images.githubusercontent.com/1755269/124814139-80eb9a80-df1a-11eb-9a6f-ed2b640b107b.png">
